src/stylesheets/core/_index.scss

```diff
@@ -0,0 +1,12 @@
+@forward "settings";
+@forward "variables";
+// @TODO is defaults necessary?
+// @forward "defaults";
```

Comment:
Defaults might not even be necessary. I'm not sure what the purpose of this is originally.